engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js

```diff
@@ -131,7 +131,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsInfoContro
         };
 
         $scope.convertMemToGB = function (memoryValue) {
-            return (memoryValue / 1024000).toFixed(2);
+            return (memoryValue/1000000).toFixed(2);
```

Comment:
I believe 1024 is correct. If we're going to switch to 1000 it should be across all the tooling. Perhaps others have different opinions?